tests/Doctrine/Tests/ORM/Proxy/ProxyFactoryTest.php

```diff
@@ -185,40 +191,42 @@ public function testFailedProxyCloningDoesNotMarkTheProxyAsInitialized()
         } catch (EntityNotFoundException $exception) {
         }
 
-        self::assertFalse($proxy->__isInitialized());
+        self::assertFalse($proxy->isProxyInitialized());
     }
 
     public function testProxyClonesParentFields()
     {
-        $companyEmployee = new CompanyEmployee();
```

Comment:
This was completely broken: the persister was returning a new instance instead of just hydrating the given object.